demo/public/assets/te-container.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
do we need all these SVGs here? they're for our textual entailment visualization

Reply:
Thanks, good catch.